include/odp/api/spec/queue_types.h

```diff
@@ -194,7 +194,8 @@ typedef struct odp_queue_capability_t {
 			  * supported when zero. */
 			uint32_t max_num;
 
-			/** Maximum queue size */
+			/** Maximum queue size. The value of zero means that
```

Comment:
You could refer to https://github.com/OpenDataPlane/odp/issues/826 in the commit message so the issue is properly tracked.

Reply:
Fixed in v2